src/lib/IlmThread/IlmThreadExport.h

```diff
@@ -3,15 +3,27 @@
 // Copyright (c) Contributors to the OpenEXR Project.
 //
 
+// See docs/SymbolVisibility.md for more discussion
+
+/// \addtogroup ExportMacros
+/// @{
 #if defined(OPENEXR_DLL)
-    #if defined(ILMTHREAD_EXPORTS)
-	    #define ILMTHREAD_EXPORT __declspec(dllexport)
-        #define ILMTHREAD_EXPORT_CONST extern __declspec(dllexport)
-    #else
-	    #define ILMTHREAD_EXPORT __declspec(dllimport)
-	    #define ILMTHREAD_EXPORT_CONST extern __declspec(dllimport)
-    #endif
+#    if defined(ILMTHREAD_EXPORTS)
```

Comment:
These #defines don't exactly mirror the IEX ones, is that intentional? There's no OPENEXR_USE_DEFAULT_VISIBILITY here. Should the two libraries follow the same structure?